### PR TITLE
Add a student

### PR DESCRIPTION
**Pull request by akshit15013**
Description : Add an Avatar of a student
Link to my website : https://akshit15013.github.io/gci18.fossasia.org/
Issue No. : #294

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -101,3 +101,7 @@
   github: saarthakchats
   image: saarthak.JPG
   twitter: SaarthakChats41
+
+- name: Akshit Goyal
+  github: akshit15013
```
Comment (pythongiant) [inline _data/students.yml]: This image is out of dimensions

Comment (akshit15013) [reply _data/students.yml]: How much is the image size Of my image?

Comment (akshit15013) [reply _data/students.yml]: I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?

**Comment by STEdirisuriya** [comment]
@akshit15013 LGTM!

**Pull request merged**